2018/002-update-channels.md

```diff
@@ -0,0 +1,123 @@
+ Title
+
+| Field  | Value      |
+|:-------|:-----------|
+| Status | Draft      |
+| Date   | 2018-07-31 |
+
+## Introduction
+
+### Problem description
+
+Automatic updates can be disruptive for a customer's application stack (ex.
+Kubernetes minor upgrade, default pod security policy introduction).
+
+Right now there is no visibility of what is inside an update and furthermore
+there is no possible selection of which updates are applied. An admin needs to
+apply all updates that come through a single channel.
+
+In order to selectively apply updates in SUSE CaaS Platform, we need different
```

Comment:
I wouldn't call that selectively apply updates. Maybe "In roder to continue applying bugfix and security updates without the need to apply intrusive, new features ..."